package.json

```diff
@@ -27,9 +27,9 @@
     "@types/mdx": "^2.0.9",
     "gray-matter": "^4.0.3",
     "mixpanel-browser": "^2.47.0",
-    "next": "13.5.6",
-    "react": "^18",
-    "react-dom": "^18",
+    "next": "14.0.3",
```

Comment:
I think at the very least, all version specifiers should allow patch upgrades (in this case: `~1.0.4` )